stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -30,18 +29,43 @@
   };
 
   private final Context mContext;
+  private final DatabaseFilesProvider mDatabaseFilesProvider;
 
-  public DatabasePeerManager(Context context) {
+  /**
+   * Constructs the object with a {@link DatabaseFilesProvider} that supplies the database files
+   * from {@link Context#databaseList()}.
+   *
+   * @param context the context
+   */
+  public DatabasePeerManager(final Context context) {
+    this(context, new DatabaseFilesProvider() {
+      @Override public Set<File> getDatabaseFiles() {
+        Set<File> databaseFiles = new HashSet<File>();
+        for (String filename : context.databaseList())
+          databaseFiles.add(new File(filename));
+        return databaseFiles;
+      }
+    });
+  }
+
+  /**
+   * @param context the context
+   * @param mDatabaseFilesProvider a database file name provider, can be null
+   */
+  public DatabasePeerManager(Context context,
+      DatabaseFilesProvider mDatabaseFilesProvider) {
```

Comment:
Yes.  Our style is to put each parameter, even the first, on its own line once we have to wrap.